packages/schematics/src/collection/library/library.spec.ts

```diff
@@ -488,11 +488,15 @@ describe('lib', () => {
           },
```

Comment:
A lot of schematics `.spec` files were changed so that the expectations match formatted text.